demo/src/lib/layout/Published.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+	export let date;
+
+	date = Intl.DateTimeFormat('en-US', {dateStyle: 'full'}).format(new Date(date));
```

Comment:
maybe it should be:
```typescript
$: dateString = Intl.DateTimeFormat('en-US', {dateStyle: 'full'}).format(new Date(date));
```

Reply:
was thinking about the reactivity dangle, but as it is rendered only once not sure that it is actually needed, wdyt?

Reply:
It's not really needed indeed... though it provides better code readability.